.config/eslint.mjs

```diff
@@ -151,5 +153,57 @@ export default [
       // see https://github.com/Splines/eslint-plugin-erb/releases/tag/v2.0.1
       reportUnusedDisableDirectives: "off",
     },
+    ignores: ["**/*.html**"],
+  },
+  {
+    // HTML linting (aside from erb_lint)
+    files: ["**/*.html", "**/*.html.erb"],
```

Comment:
@fosterfarrell9 With regards to linting of the `classification.html.erb` file: for me, the only thing that was not correctly configured was the spacing of VSCode's rulers (soft gray lines in the background). In the latest commit, I now set `"editor.indentSize": 2` and `"editor.tabSize": 2` explicitly for HTML, JS and ERB files. It might still be that the spacing for this file is set not correctly (don't know why, maybe a VSCode bug), then just run the `Detect Indentation from Content` command once for this file and it should work fine.

Was there something else that was not properly linted for you in this file?

Reply:
No, I guess that was it.